.packit.yaml

```
@@ -42,6 +42,13 @@ actions:
     - "python3 setup.py --version"
 
 jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+    - fedora-all
+    - epel-8
+    - epel-9
+  
```

And if possible change the targets to the format: 
```suggestion
    targets:
    - targets: [fedora-all, epel-8, epel-9]
```
so our lint does not cry at lout

---

If you do this manually, installing `pre-commit` to the repository helps to identify all the issues before a push.

---

sure, will update it :) (it was done via a script)